client/go/cmd/login.go

```diff
@@ -93,3 +94,22 @@ func newLoginCmd(cli *CLI) *cobra.Command {
 		},
 	}
 }
+
+func confirm(question string) bool {
+	for {
+		var answer string
+
+		log.Printf("%s [Y/n] ", question)
```

Comment:
Use `fmt.Fprintf(cli.Stdout, ...)` instead of `log.Printf()`. That way we avoid a newline after the prompt.
At least for me that ensures that I catch the action that is required. When the prompt is on an empty line I just wait for more output.